File: week1/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int get_height(void);

int main(void)
{
    int rows = get_height();

    for (int i = 1; i <= rows; i++)
    {
        for (int j = 1; j <= rows - i; j++)
        {
            printf(" ");
        }

        for (int j = 1; j <= i; j++)
        {
            printf("#");
        }

        for (int j = 1; j <= 2; j++)
        {
            printf(" ");
        }

        for (int j = 1; j <= i; j++)
        {
            printf("#");
        }

        printf("\n");
    }
}

int get_height(void)
{
    int height;
    do
    {
        height = get_int("Height: ");
    }
    while (height <= 0 || height > 8);

    return height;
}

File: week6/sentimental-mario-more/mario.py
def main():
    height = get_height()
    for i in range(1, height + 1):
        print(" " * (height - i) + "#" * i + "  " + "#" * i)


def get_height():
    while True:
        try:
            height = int(input("Height: "))
            if 1 <= height <= 8:
                return height
        except ValueError:
            pass


main()

